src/raven.js

```diff
@@ -870,4 +869,13 @@ function afterLoad() {
         Raven.config(RavenConfig.dsn, RavenConfig.config).install();
     }
 }
+
+function urlencode(o) {
+    var pairs = [];
+    each(o, function(key, value) {
+        pairs.push(encodeURIComponent(key)+'='+encodeURIComponent(value));
```

Comment:
Eh, I feel with all the global variables etc, can just wait until we do the refactor.